src/redturtle/agidtheme/browser/templates/sights_renderer.pt

```diff
@@ -8,7 +8,7 @@
     <body>
         <metal:macro define-macro="collection-tile-macro"
                      tal:define="helper_view nocall:context/@@collection_tile_helpers;
-                                 showCarousel python: 'carousel' in view.data.get('css_class', '');
+                                 showCarousel python:'carousel' in (view.data.get('css_class') or '');
```

Comment:
If I remember correctly, the new proposed `(view.data.get('css_class') or '')` and the old `view.data.get('css_class', '')` actually behave in the same way and return the same values. Am i missing something? Anyway, it's ok for me either way.

Reply:
yes, but the old one will breaks if css_class is empty